### PR TITLE
SQL Intro cleanup

### PR DESCRIPTION
**Pull request by ethanwhite**
1. Integrate instruction on importing, types, and primary keys into initial introduction to databases
2. Other minor cleanup

### CODE REVIEW

File: materials/databases-intro-sql.md [hunk 1]
```diff
@@ -37,20 +37,44 @@ language: SQL
...
 ### Relational databases
 
 * Data is stored in tables.
     * One table per type of data
     * Tables can be linked together to combine information.
+
+> Open SQLite Manager
+> Walk through importing surveys table
+
+1. Download https://ndownloader.figshare.com/files/2292172
```
Comment (brymz) [inline materials/databases-intro-sql.md]: Do you want this as a link?

File: materials/databases-intro-sql.md [hunk 2]
```diff
@@ -37,20 +37,44 @@ language: SQL
...
+    * We have to define the types in advance.
+    * Types include
+        * Integer
+        * Text
+        * Decimal/Double/Real/Float
+    * Types are highly configurable for when space is limiting
+        * Maximum value of integers
+        * Maximum length of text
+        * How many values before and after the decimal place
+9. Select `recordID` as the `Primary Key` and click `OK`.
```
Comment (brymz) [inline materials/databases-intro-sql.md]: `record_id`

File: materials/databases-intro-sql.md (hunk 1 above)
Comment (ethanwhite) [inline materials/databases-intro-sql.md]: Links in the materials are normally used to open something on a separate machine so they need to be written out so they can be typed in.

Comment (ethanwhite) [inline materials/databases-intro-sql.md]: Though actually we might as well make it a link as well just in case.

**Comment by ethanwhite** [comment]
Done. Thanks.

**Pull request merged**